connectionchecker/connection_checker.go

```diff
@@ -206,3 +209,20 @@ func (cc *ConnectionChecker) checkProxySQLConnection(ctx context.Context, dsn st
 
 	return &res
 }
+
+func (cc *ConnectionChecker) checkExternalConnection(ctx context.Context, uri string) *agentpb.CheckConnectionResponse {
+	var res agentpb.CheckConnectionResponse
+
+	resp, err := http.Get(uri)
```

Comment:
Does it make sense to use the context like you did [here](https://github.com/percona/pmm-managed/pull/659/files#diff-efe2a6cfe03bd019245b4fa72eb3a366a044451536644d616929aff48cc0da91R179)?

Reply:
Yeah, tricky point. Chenged.